01.general/11.https-filtering/docs.en.md

```diff
@@ -11,71 +11,70 @@ visible: true
+Using HTTPS is strictly advantageous since encrypted traffic is protected from ‘eavesdropping’ by third parties, and we can only welcome this. HTTPS spread has been on the rise in recent years, mostly because it is being [encouraged by Google](https://webmasters.googleblog.com/2014/08/https-as-ranking-signal.html) and also due to the emergence of a free certification center [Let’s Encrypt](https://en.wikipedia.org/wiki/Let's_Encrypt).
 
 The diagram below describes the difference between plain HTTP protocol and secure HTTPS protocol.
 
 ![What is HTTPS](https://cdn.adguard.com/public/Adguard/Blog/https/what_is_https.png)
 
 ### What is a security certificate?
 
-Simply put, HTTPS means data encryption. But there’s still a problem: how can you be sure you’ve established an encrypted connection with exactly the website you intended to? That’s where security certificates come into play. The certificate is proof that the website is actually what it tells you it is. If a website doesn’t have such certificate, or the certificate contains incorrect information, the browser won’t let you establish a secure connection. It is important that the certificate a website uses was issued by a certification authority (CA) trusted by your browser. Such CA guarantees that the SSL certificate is, indeed, issued to the website’s owner.
+Simply put, HTTPS means data encryption. But there’s still a problem: how can you be sure you’ve established an encrypted connection with exactly the website you intended to? That’s where security certificates come into play. The certificate is proof that the website is actually what it tells you it is. If a website doesn’t have such a certificate, or if the certificate contains incorrect information, the browser won’t let you establish a secure connection. It is important that the certificate a website uses is issued by a certification authority (CA) trusted by your browser. Such a CA guarantees that the SSL certificate is, indeed, issued to the website’s owner.
 
 ### Why does AdGuard need to be able to filter HTTPS?
 
-The thing is, more and more websites, blogs, social media, etc. switch to HTTPS every day. And following blogs and websites, more ad networks switch to HTTPS too, because it becomes necessary to display ads on the HTTPS-based website. Here are few examples of popular websites, where you can’t remove ads without filtering the HTTPS: youtube.com, facebook.com, twitter.com.
+The thing is, more and more websites, blogs, social media, etc., switch to HTTPS every day. And following blogs and websites, more ad networks are switching to HTTPS too, because it becomes necessary to display ads on HTTPS-based websites. Here are a few examples of popular websites where you can’t remove ads without filtering the HTTPS: youtube.com, facebook.com, and twitter.com.
```

Comment:
```suggestion
The thing is, as more and more websites, blogs, social media, and others switch to HTTPS, more ad networks follow them to HTTPS as well, because it becomes necessary to display ads on HTTPS-based websites. Here are a few popular websites where you can’t remove ads without HTTPS filtering: youtube.com, facebook.com, and twitter.com.
```